src/packages/client/src/__tests__/integration/happy/groupBy/test.ts

```diff
@@ -1,190 +1,159 @@
 import { getTestClient } from '../../../../utils/getTestClient'
 
 let prisma
+describe('groupBy', () => {
+  beforeAll(async () => {
+    const PrismaClient = await getTestClient()
+    prisma = new PrismaClient()
+  })
 
-beforeAll(async () => {
-  const PrismaClient = await getTestClient()
-  prisma = new PrismaClient()
-})
+  afterAll(async () => {
+    await prisma.$disconnect()
+  })
 
-afterAll(async () => {
-  await prisma.$disconnect()
-})
+  test('email', async () => {
+    const user = await prisma.user.groupBy({
+      by: ['email'],
+    })
 
-test('groupBy email', async () => {
-  const user = await prisma.user.groupBy({
-    by: ['email'],
+    expect(user).toMatchSnapshot()
```

Comment:
Do we have a preference between inline snapshots or external snaps?

Reply:
inline is nice when small because it's easy to read and verify/review but gets annoying when large.
Locally this extension makes it possible to quickly look for snapshots by hovering the line https://marketplace.visualstudio.com/items?itemName=Orta.vscode-jest

Reply:
I just changed this because the file was becoming pretty unwieldy. Generally, I prefer inline snapshots